Casks/testcontainers-cloud-desktop.rb

```diff
@@ -38,15 +38,18 @@
   "
 
   uninstall delete: [
-              "~/Library/Caches/AtomicJar",
```

Comment:
Let's not touch now-deprecated TCC cask. WDYT?

Reply:
This way people who already installed the old cask will get the fix too